app/src/main/java/org/cirdles/topsoil/app/TopsoilMainWindow.java

```diff
@@ -373,4 +376,27 @@ void emptyTable() {
         getCurrentTable().ifPresent(TsvTable::clear);
     }
 
+    @FXML
+    void openDocumentationLink() {
+        try {
+            if (Desktop.isDesktopSupported()) {
+                Desktop.getDesktop().browse(new URI("https://github.com/CIRDLES/Topsoil/wiki"));
+            }
+        } catch (IOException | URISyntaxException e) {
+            LOGGER.error(null, e);
+        }
+
+    }
+
+    @FXML
+    void openHelpLink() {
```

Comment:
No reason to duplicate this code (especially if you realize just how complex this logic can be).